platform_cli/helpers.py

```
@@ -11,6 +11,50 @@
+    try:
+        result = stdout_call(f"git diff --name-only {base_branch}...HEAD", cwd=cwd, abort=False)
+        if not result.strip():
+            return []
+
+        workspace = Path(cwd) if cwd else Path.cwd()
+        return [
+            (workspace / line.strip()).resolve()
+            for line in result.strip().splitlines()
+            if line.strip()
+        ]
+    except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)